api4/status.go

```diff
@@ -40,7 +40,19 @@ func getUserStatus(c *Context, w http.ResponseWriter, r *http.Request) {
 func getUserStatusesByIds(c *Context, w http.ResponseWriter, r *http.Request) {
 	userIds := model.ArrayFromJson(r.Body)
 
+	isUserIdsInvalid := false
+
 	if len(userIds) == 0 {
+		isUserIdsInvalid = true
+	}
+
+	for _, userId := range userIds {
+		if len(userId) != 26 {
+			isUserIdsInvalid = true
```

Comment:
I think we can simplify and return directly here. No need for the additional variable. We can stop and return as soon as we find something invalid.

Reply:
my only reason was to avoid code duplication and improve readability a bit. Do you want to avoid the variable, because this function is often called?
Or should I treat it as a general rule for future code changes?

Reply:
My reasoning is to return early to keep the whole logic easier to understand. 
If we get an empty slice there's no need to go any further. It's an unrecoverable error.

Avoiding code duplication is great but in this specific case I don't think we are gaining much on that front.